mappings/common/common_test.go

```diff
@@ -0,0 +1,46 @@
+package common
+
+import (
+	"database/sql"
+	"fmt"
+	"github.com/bitcoin-sv/spv-wallet/engine"
+	"github.com/bitcoin-sv/spv-wallet/engine/datastore/customtypes"
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"time"
+)
+
+func TestMapToContract_NewlyCreatedRecord(t *testing.T) {
+	currentTimestamp := time.Now().UTC()
+	engineModel := engine.Model{
+		CreatedAt: currentTimestamp,
+		UpdatedAt: currentTimestamp,
+	}
+
+	commonModel := MapToContract(&engineModel)
+	fmt.Printf("Struct: %+v", commonModel)
```

Comment:
First of all I'm not convinced that we want to print it.
Beside that, I would rather suggest using `t.Logf` instead of `fmt.Printf` because it will add the context of the line in file which prints the message, so it's easier to find it.
But still - IMHO it's not needed.

Reply:
Agree.
Fixed.